ts/IdentificationModal.tsx

```diff
@@ -214,7 +214,9 @@ class IdentificationModal extends React.PureComponent<Props, State> {
         this.props.identificationState.kind === "started")
     ) {
       this.maybeTriggerFingerprintRequest({
-        updateBiometrySupportProp: this.props.appState === "active"
+        updateBiometrySupportProp:
+          prevProps.appState === "background" &&
```

Comment:
```suggestion
          prevProps.appState !== "active" &&
```

Remember on iOS we also have `inactive` state.

Reply:
You are right. Changed.